cypress/e2e/notifications.spec.ts

```diff
@@ -0,0 +1,112 @@
+import * as api from "../api"
+import {
+  E2E_EMAIL_ADMIN,
+  E2E_EMAIL_COLLAB,
+  E2E_EMAIL_TEST_SITE,
+} from "../fixtures/constants"
+import {
+  genRandomString,
+  getNotificationsButton,
+  visitE2eEmailTestRepo,
+} from "../utils"
+
+const NEW_COMMENTS_BG_COLOR = "rgb(235, 248, 255)"
+const EMPTY_NOTIFICATIONS_TEXT = "There are no notifications for display."
+const getReviewRequestedNotifText = (email: string) =>
+  `${email} has sent you a review request.`
+
+const getNewNotifBadge = () =>
+  getNotificationsButton().within(() => {
+    cy.get(".chakra-avatar").within(() => {
+      cy.get("div")
+    })
+  })
+
+describe("notifications", () => {
+  let reviewId: number
+  beforeEach(() => {
+    cy.setEmailSessionDefaults("Email admin")
+    cy.setupDefaultInterceptors()
+    visitE2eEmailTestRepo()
+  })
+  before(() => {
+    cy.createEmailUser(
+      E2E_EMAIL_COLLAB.email,
+      "Email admin",
+      "Email admin",
+      E2E_EMAIL_TEST_SITE.name
+    )
+    // NOTE: Need to set permissions as order is `before` -> `beforeEach`
+    visitE2eEmailTestRepo()
+    // NOTE: Mark all notifications as read as email collab.
+    // This is because previous tests may have created notifications.
+    api.markNotificationsAsRead()
+
+    cy.setEmailSessionDefaults("Email admin")
+    cy.setupDefaultInterceptors()
+    api.closeReviewRequests()
+    // NOTE: Mark all notifications as read as email collab.
+    // This is because previous tests may have created notifications.
+    api.markNotificationsAsRead()
```

Comment:
sanity check here, is the duplication of the lines 
```
 // NOTE: Mark all notifications as read as email collab.
    // This is because previous tests may have created notifications.
    api.markNotificationsAsRead()
```
intended? There are no additional tests that occur within the duplicated 
`api.markNotificationsAsRead()` right?

Reply:
wait my comment is wrong but the duplication **is intended**. in short, we're acting as 2 users (teh collab and the admin) so we clear notifs for both. 

the backend clears notifs through the session data, which is generated from the request's cookies. because we set cookies on the `setUpXxxDefaults` function, we need to do it twice.

Reply:
updated comment for clarity

Reply:
updated comment for clarity